### PR TITLE
Resolve esoteric IDs in the fuel category

### PR DESCRIPTION
**Pull request by patstirling**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Fuel

**What?**

Resolve esoteric IDs in the fuel category

**Why?**

Provide better comparability across EFs

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -46,10 +46,10 @@ Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_gasoline-di
...
+Energy,Fuel,fuel_type_aviation_fuel_petroleum,Aviation fuel (petroleum),kg-CO2e/l,2.33048,,BEIS,2021,GB,2021/07/23,Emission intensity of aviation spirit fuel for piston-engined aircraft - a high octane petrol (aka AVGAS). The fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,combustion
+Energy,Fuel,fuel_type_aviation_fuel_kerosene,Aviation turbine fuel (kerosene),kg-CO2e/l,2.54514,,BEIS,2021,GB,2021/07/23,Emission intensity of aviation turbine fuel for turbo-prop aircraft and jets (aka jet fuel). Similar to kerosene used as a heating fuel but refined to a higher quality. The fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:52]: Aviation turbine is what is being noted by the source...

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -271,13 +271,13 @@ Computing,Memory,memory-provider_azure-region_west_us,AZURE (west-us) memory,kg-
...
-Energy,Fuel,fuel_type_biodiesel_me_used_cooking_oil,Biodiesel ME (From used cooking oil),kg-CO2e/l,0.16751,,BEIS,2021,GB,2021/07/23,Emission intensity of biodiesel ME (from used cooking oil). Bioenergy conversion factors should be used for the combustion of fuels produced from recently living sources (such as trees) at a site or in an asset under the direct control of the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,combustion
+Energy,Fuel,fuel_type_biodiesel_hvo,Biodiesel HVO,kg-CO2e/l,0.03558,,BEIS,2021,GB,2021/07/23,Emission intensity of hydrogenated vegetable oil (HVO) biodiesel. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:274]: Why do we remove the decription?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1055,7 +1055,7 @@ Transport,Vehicle,passenger_vehicle-vehicle_type_large_motorbike-fuel_source_pet
...
-Energy,Fuel,fuel_type_petrol_100_%_mineral,Petrol 100% mineral,kg-CO2e/l,2.33969,,BEIS,2021,GB,2021/07/23,Emission intensity of 100% mineral petrol. The fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,combustion
+Energy,Fuel,fuel_type_petrol,Petrol,kg-CO2e/l,2.33969,,BEIS,2021,GB,2021/07/23,Emission intensity of 100% mineral petrol. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1058]: why do we remove the description?

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -46,10 +46,10 @@ Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_gasoline-di
...
+Energy,Fuel,fuel_type_aviation_fuel_kerosene,Aviation turbine fuel (kerosene),kg-CO2e/l,2.62632,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation fuel (kerosene)/Jet A1. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,combustion
+Energy,Fuel,fuel_type_aviation_fuel_petroleum,Aviation fuel (petroleum),kg-CO2e/l,2.3123049,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation gasoline. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,combustion
```
[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:50]: Other fuel EFs (e.g. the BEIS ones) which are used for transport don't specify this, so to provide standardisation decided to normalise this EF with the others and for the outlier (stationary) provide a point of difference

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:52]: If you look at the two sources here, they are both reporting two things - kerosene-based aviation fuel (aka AVTUR, jet fuel or turbine fuel - https://en.wikipedia.org/wiki/Aviation_fuel#Jet_fuel, https://en.wikipedia.org/wiki/Jet_fuel) and petrol-based aviation fuel (aka Avgas https://en.wikipedia.org/wiki/Aviation_fuel#Avgas, https://en.wikipedia.org/wiki/Avgas). I'll correct to "(petrol)" - this should make it clear what the two are, and standardise them

File: OpenEmissionFactorsDB.csv (hunk 4 above)
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv]: let us add it to the name then?

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:274]: The description is designed to help the user select the appropriate EF - in these instances the text is methodology-oriented, which we don't need to include (there is a much larger process to go through ragarding this!)

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:1058]: see above

**Review by AnastasiaLobanova — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv (hunk 4 above)
[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: yep it is (line 408): "Diesel (stationary industrial use)"

[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv]: but the "transport" is not in the name, right?

[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: Good point!

[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: Wait yes they are in the name

**Review by maryarbab — APPROVED** [review]
Looks great, Pat! Thank you so much for making the effort to add the "use" sub-category!

**Review by AnastasiaLobanova — APPROVED** [review]

**Pull request merged**